tt_helpers.go

```diff
@@ -0,0 +1,121 @@
+package tupleconv
+
+import "errors"
+
+// ErrInvalidSpaceFmt is an error of the space format.
+var ErrInvalidSpaceFmt = errors.New("invalid space format")
+
+const (
+	kType       = "type"
+	kIsNullable = "is_nullable"
+)
+
+const (
+	defaultIgnoreChars   = ""
+	defaultDecSeparators = "."
+	defaultNullValue     = ""
+)
+
+// MappersTable can return a Mapper[string] based on the type name.
+type MappersTable interface {
```

Comment:
Why is it in the public API?

Reply:
This entity was used in internal tests, was deleted.